blueocean-dashboard/src/main/resources/jenkins/plugins/blueocean/dashboard/Messages_de.properties

```diff
@@ -0,0 +1,64 @@
+Activity=Aktivit\u00E4ten
```

Comment:
![screenshot from 2016-10-25 11-49-05](https://cloud.githubusercontent.com/assets/596701/19681288/1a94a3ae-9aa9-11e6-86a3-1a21907b4e23.png)

IMHO that needs to stay in Plural